test/lint.js

```diff
@@ -1,92 +1,26 @@
-var fs = require('fs');
-var path = require('path');
-var Ajv = require('ajv');
-var ajv = new Ajv({ $data: true, allErrors: true });
-var dictPaths = ['api', 'css', 'l10n'];
-var hasErrors = false;
+var fs = require('fs')
```

Comment:
It's not fair, you have changed a code style 😢

Reply:
Apologies. That was my editor default settings. Do you want me to bring back the semicolons?